src/components/ListTemplate/ColumnTemplates/DefaultColumn.js

```diff
@@ -44,6 +48,26 @@ export const DefaultColumn = memo(({
       {content}
     </AppIcon>
   );
+
+  if (!iconKey) {
+    return content;
+  }
+
+  return (
+    <>
+      {showAsHotLink ? (
+        <Button
+          data-test-profile-link
+          buttonStyle="link"
+          marginBottom0
+          to={`/settings/data-import/${entityName}/view/${recordId}`}
+          buttonClass={sharedCss.cellLink}
+        >
```

Comment:
Consider moving this to a separate variable
```suggestion
        {hotLink}
```